src/rand.rs

```diff
@@ -109,13 +113,43 @@ impl<T> RandomlyConstructable for T where T: self::sealed::RandomlyConstructable
 /// `fill()` once at a non-latency-sensitive time to minimize latency for
 /// future calls.
 #[derive(Clone, Debug)]
-pub struct SystemRandom(());
+pub struct SystemRandom;
 
 impl SystemRandom {
     /// Constructs a new `SystemRandom`.
     #[inline(always)]
-    pub fn new() -> Self {
-        Self(())
```

Comment:
It is intentional that we reserve the right to add fields to `SystemRandom` and we reserve the right for `SystemRandom` to not be `const`-constructable. We will need to wait until the FIPS project is further along before we can revisit this.

Reply:
Understood. Thank you.